README.md

```diff
@@ -55,8 +55,28 @@ download and unarchive the source tarball (Appium-Python-Client-X.X.tar.gz).
     - It generates readable changelog
 - Setup
     - `pip install --user pipenv`
-    - `pipenv lock --clear`
-    - `pipenv install --dev --system`
+    - `python -m pipenv lock --clear`
+       - If you have below error, refer [this link](https://github.com/pypa/pipenv/issues/187) to solve it.
+          ```
+          Locking [dev-packages] dependencies…
```

Comment:
Too much? @mykola-mokhnach

Reply:
yeah, it would be enough to just keep the error message itself: `Locking Failed! unknown locale: UTF-8`

Reply:
Thanks, it's smart.